src/comb_filter.rs

```diff
@@ -19,28 +28,89 @@ pub enum Error {
     InvalidValue { param: FilterParam, value: f32 }
 }
 
+
+
 impl CombFilter {
     pub fn new(filter_type: FilterType, max_delay_secs: f32, sample_rate_hz: f32, num_channels: usize) -> Self {
-        todo!("implement")
+        let filter_params = (1.0, 0.0);
+        let delay_time = (max_delay_secs * sample_rate_hz) as usize;
+        let delay_line = vec![0.0; delay_time];
```

Comment:
Have you added tests for more than 1 or 2 channels? What will happen when there are 6 channels for example? I might suggest allocating multiple delay lines for each channel or allocating enough space for each individual channel in one buffer. Otherwise it might end up overwriting existing data.